tools/osbuild-image-test

```diff
@@ -503,72 +625,37 @@ def main():
         default="manifest-db",
         help="path to the database folder"
     )
+    parser.add_argument(
+        '--command',
+        type=str,
+        choices=[
+            "run",
+            "diff"
+        ],
+        default="run")
+    parser.add_argument(
+            "--item1",
```

Comment:
nitpick: I would suggest to call his and `item2` in a more self-explanatory way, such as `base` and `new`.

Reply:
Now the files to diff are positional arguments, so naming convention is fixed